src/components/templates/ReactionPage/ReactionPage.tsx

```diff
@@ -21,15 +21,16 @@ const wantedReactionsSelector = SHOW_EMOJI_IN_REACTION_PAGE
 
 // @debt pass venue through the props
 export const ReactionPage: React.FC = () => {
-  const venueId = useVenueId();
-  const { currentVenue } = useConnectCurrentVenueNG(venueId);
+  const spaceSlug = useSpaceParams();
+  const { space } = useSpaceBySlug(spaceSlug);
+  const venueId = space?.id;
 
   // @debt this is very similar to the query in src/hooks/reactions.tsx, but that filters by createdAt > now
   useFirestoreConnect(
-    currentVenue
+    venueId
```

Comment:
👍 
This check looks like an improvement to the original logic. `venueId` might have come from URL, but `currentVenue` still missing.

Using `space?.id` is better. Might have not bothered with the rename though... check above comment.